.github/dependabot.yml

```diff
@@ -3,7 +3,8 @@ updates:
   - package-ecosystem: gomod
     directory: /
     schedule:
-      interval: daily
+      interval: weekly
+      day: sunday
```

Comment:
Might be worth doing this on Friday so that PRs are merged by EOW and then we will have the CI scheduled runs to test the new changes.
```
    schedule:
      interval: "weekly"
      day: friday
      time: "00:00"
      timezone: "Europe/London"
```